package.json

```
@@ -269,6 +269,12 @@
           "additionalProperties": false,
           "markdownDescription": "Whether to display Quick Fix actions to disable rules via `noqa` suppression comments."
         },
+        "ruff.showSyntaxErrors": {
+          "default": true,
+          "markdownDescription": "Controls whether Ruff should show syntax error diagnostics.",
```

To align with other descriptions
```suggestion
          "markdownDescription": "Whether to show syntax error diagnostics.",
```